themes.txt

```diff
@@ -78,7 +78,7 @@ github.com/bep/docuapi/v2
 github.com/bjacquemet/personal-web
 github.com/blankoworld/hugo_theme_adam_eve
 github.com/brycematheson/allegiant
-github.com/budparr/gohugo-theme-ananke
```

Comment:
Switching

Reply:
I should have given context. We switched for the `theNewDynamic` which is the org of the company I'm a partner of. Founder is @budparr who created the theme. This is not a fork of the original theme, just its new home. https://github.com/theNewDynamic/gohugo-theme-ananke

We were relying on github redirect for now, but I noticed that since the change the hugoThemeSiteBuilder has not been pulling the latest info of the theme, so I'm here to address that.